### PR TITLE
Ports Suit Storage Unit Resprite TG(#63291) Shiptest(#713)

### PR DESCRIPTION
**Pull request by KathrinBailey**
## About The Pull Request
• Ports https://github.com/tgstation/tgstation/pull/63291 from TG, ported from Shipstation.
• Ports https://github.com/shiptest-ss13/Shiptest/pull/713 from Shipstation, courtesy of Retlaw34!

## Why It's Good For The Game
They look so much better than the flat-ass pancake sprites we had before.
image
image
**Above are the fully updated sprites**

## Changelog
:cl: retlaw34
imageadd: Suit storage unit sprites ported from Shipstation!
/:cl:

### CODE REVIEW

**Comment by Azarak** [comment]
I see interaction logic was changed. Can you still pry them open when they're unpowered?

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -289,10 +292,12 @@
 			if (item_to_dispense)
 				vars[choice] = null
 				try_put_in_hand(item_to_dispense, user)
+				update_icon()
```
Comment (Azarak) [inline code/game/machinery/suit_storage_unit.dm:295]:
```suggestion
				update_appearance()
```

File: code/game/machinery/suit_storage_unit.dm [hunk 2]
```diff
@@ -289,10 +292,12 @@
...
 			else
 				var/obj/item/in_hands = user.get_active_held_item()
 				if (in_hands)
 					attackby(in_hands, user)
+				update_icon()
```
Comment (Azarak) [inline code/game/machinery/suit_storage_unit.dm:300]:
```suggestion
				update_appearance()
```

**Review by Azarak — COMMENTED** [review]
`update_appearance()` is the proper proc for updating appearance. You can pass flags to it if you want to restrict it only updating some parts of the appearance, but this isn't nessecary in most cases

**Comment by atakiya** [comment]
> I see interaction logic was changed. Can you still pry them open when they're unpowered?

Yes you can, however, in some instances you cant get the items out when the doors are open, and power is lost (by for example, initiating the cleaning, then power being lost before it finishes).
That's beside the fact it will drop all items out if power is lost, which is a ????? in itself.

**Pull request merged**